tests/Database/DatabaseSeederTest.php

```diff
@@ -46,6 +46,8 @@ public function testCallResolveTheClassAndCallsRun()
         $child->shouldReceive('setContainer')->once()->with($container)->andReturn($child);
         $child->shouldReceive('setCommand')->once()->with($command)->andReturn($child);
         $child->shouldReceive('__invoke')->once();
+        $command->shouldReceive('getOutput')->once()->andReturn($output);
+        $output->shouldReceive('writeln')->once()->andReturn('foo');
```

Comment:
Does writeln really return 'foo' in real usage?

Reply:
nope, fair point. I'll remove both of these calls